doc/supervised.rst

```diff
@@ -231,13 +235,14 @@ the sum of probability of being correctly classified:
     nca = NCA(max_iter=1000)
     nca.fit(X, Y)
 
-.. topic:: References:
+.. rubric:: References
+
+
+.. container:: hatnote hatnote-gray
```

Comment:
it would be nice it there was a way to define a custom rubric or something so we only need to write something like 
``.. rubric:: References``
and the appropriate CSS style would be automatically applied

Reply:
(no big deal if there is no easy way to do this)

Reply:
After researching for while, there is not a clear path to this at all. Also, I think that even if its possible all `.. rubric::` stuff will have the same CSS, including the Examples, so I guess its not a good idea to have this CSS globally, because we don't want grey boxes everywhere, just in the references.